src/open_inwoner/accounts/tests/test_auth_2fa_sms.py

```diff
@@ -0,0 +1,535 @@
+import datetime
+import time
+from unittest.mock import ANY, patch
+
+from django.conf import settings
+from django.core.signing import TimestampSigner
+from django.test import TestCase, override_settings
+from django.test.utils import freeze_time as dj_freeze_time
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.http import urlencode
+from django.utils.translation import gettext as _
+
+from freezegun import freeze_time
+from timeline_logger.models import TimelineLog
+
+from open_inwoner.configurations.models import SiteConfiguration
+from open_inwoner.utils.logentry import LOG_ACTIONS
+from open_inwoner.utils.validators import format_phone_number
+
+from ..gateways import GatewayError
+from .factories import UserFactory
+
+signer = TimestampSigner()
+
+
+@override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
+class TestSMSVerificationLogin(TestCase):
+    def setUp(self):
+        self.user = UserFactory(email="ex@example.com", password="secret")
+
+        self.config = SiteConfiguration.get_solo()
+        self.config.login_2fa_sms = True
+        self.config.login_allow_registration = True
+        self.config.save()
+
+    @patch("open_inwoner.accounts.gateways.gateway.send")
+    def test_regular_login_with_2fa_disabled(self, mock_gateway_send):
+        self.config.login_2fa_sms = False
+        self.config.save()
+
+        response = self.client.get(reverse("login"))
+        mock_gateway_send.assert_not_called()
+
+        response = self.client.post(
+            reverse("login"),
+            data={
+                "next": "",
+                "username": self.user.email,
+                "password": "secret",
+            },
+            follow=True,
+        )
+
+        self.assertNotContains(response, "Account verificatie")
+        self.assertRedirects(response, reverse("pages-root"))
```

Comment:
Is this assertRedirect correct if we use `follow=True` above?

Also this test relates to my other comment about the authentication backend and 2fa?